maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java

```diff
@@ -43,7 +43,56 @@ void givenApiFindHardshipResponseWithSection_whenFindHardshipResponseToHardshipD
         softly.assertThat(actual)
                 .usingRecursiveComparison()
                 .ignoringCollectionOrder()
+                .ignoringFields("incomeEvidence.applicantIncomeEvidenceList",
+                        "incomeEvidence.partnerIncomeEvidenceList",
+                        "incomeEvidence.extraEvidenceList",
+                        "full.sectionSummaries",
+                        "initial.sectionSummaries")
                 .isEqualTo(expected);
+
+        compareIncomeEvidenceFields(actual.getIncomeEvidence().getApplicantIncomeEvidenceList(),
+                expected.getIncomeEvidence().getApplicantIncomeEvidenceList());
+        compareIncomeEvidenceFields(actual.getIncomeEvidence().getPartnerIncomeEvidenceList(),
+                expected.getIncomeEvidence().getPartnerIncomeEvidenceList());
+        compareExtraEvidenceFields(actual.getIncomeEvidence().getExtraEvidenceList(),
+                expected.getIncomeEvidence().getExtraEvidenceList());
+        compareSectionSummariesFields(actual.getFull().getSectionSummaries(),
+                expected.getFull().getSectionSummaries());
+        compareSectionSummariesFields(actual.getInitial().getSectionSummaries(),
+                expected.getInitial().getSectionSummaries());
```

Comment:
Do we need this? It seems to work without ignoring the fields

Reply:
The tests are passing in local, but keep failing in the circleci build.

Reply:
Fixed.